sdk/confidentialledger/Azure.Storage.ConfidentialLedger/README.md

```diff
@@ -42,69 +40,241 @@ export AZURE_TENANT_ID="tenant id"
 ```
 Then, `DefaultAzureCredential` will be able to authenticate the `ConfidentialLedgerClient`.
 
-Constructing the client also requires your Confidential Ledger's URL and id, which you can get from the Azure CLI or the Azure Portal. When you have retrieved those values, please replace instances of `"my-ledger-id"` and `"https://my-ledger-url.confidential-ledger.azure.com"` in the examples below
+Constructing the client also requires your Confidential Ledger's URL and id, which you can get from the Azure CLI or the Azure Portal.  When you have retrieved those values, please replace instances of `"my-ledger-id"` and `"https://my-ledger-url.confidential-ledger.azure.com"` in the examples below
 
-Because Confidential Ledgers use self-signed certificates securely generated and stored in an SGX enclave, the certificate for each Confidential Ledger must first be retrieved from the Confidential Ledger Identity Service.
+Because Confidential Ledgers use self-signed certificates securely generated and stored in an SGX enclave, the certificate for each Confidential Ledger  must first be retrieved from the Confidential Ledger Identity Service.
 
+```C# Snippet:GetIdentity
+Uri identityServiceUri = "<the identity service uri>";
+var identityClient = new ConfidentialLedgerIdentityServiceClient(identityServiceUri, new Identity.DefaultAzureCredential());
 
-TODO
+// Get the ledger's  TLS certificate for our ledger.
+string ledgerId = "<the ledger id>"; // ex. "my-ledger" from "https://my-ledger.eastus.cloudapp.azure.com"
+Response response = identityClient.GetLedgerIdentity(ledgerId);
+
+// extract the ECC PEM value from the response.
+var eccPem = JsonDocument.Parse(response.Content)
+    .RootElement
+    .GetProperty("ledgerTlsCertificate")
+    .GetString();
+
+// construct an X509Certificate2 with the ECC PEM value.
+X509Certificate2 ledgerTlsCert = new X509Certificate2(Encoding.UTF8.GetBytes(eccPem));
+```
+
+Now we can construct the `ConfidentialLedgerClient` with a transport configuration that trusts the `ledgerTlsCert`.
+
+```C# Snippet:CreateClient
+// Create a certificate chain rooted with our TLS cert.
+X509Chain certificateChain = new();
+certificateChain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
+certificateChain.ChainPolicy.RevocationFlag = X509RevocationFlag.ExcludeRoot;
+certificateChain.ChainPolicy.VerificationFlags = X509VerificationFlags.AllowUnknownCertificateAuthority;
+certificateChain.ChainPolicy.VerificationTime = DateTime.Now;
+certificateChain.ChainPolicy.UrlRetrievalTimeout = new TimeSpan(0, 0, 0);
+certificateChain.ChainPolicy.ExtraStore.Add(ledgerTlsCert);
+
+// Define a validation function to ensure that the ledger certificate is trusted by the ledger identity TLS certificate.
+Func<HttpRequestMessage, X509Certificate2, X509Chain, SslPolicyErrors, bool> certValidationCheck;
```

Comment:
Should we consider using a local function instead of the `Func` definition?  That can often reduce visual clutter and be easier to read.

Reply:
I went back and forth on this, so I'll lean in your suggested direction.